tests/frontends/torch/test_torch_hardsigmoid.py

```diff
@@ -0,0 +1,35 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from hidet import ops
+import torch
+
+from hidet.testing import check_torch_unary
+from hidet.graph.frontend.torch.register_modules import HidetHardsigmoid
+
+
+@pytest.mark.parametrize("shape", [[10, 10, 10], [15, 15, 15]])
+def test_flatten(shape):
+    hardsigmoid = HidetHardsigmoid(torch.nn.Hardsigmoid())
+    check_torch_unary(
+        shape,
+        lambda x: torch.nn.functional.hardsigmoid(x),
+        lambda x: hardsigmoid(x),
+        dtype='float32',
+        atol=1e-5,
+        rtol=1e-5,
+    )
```

Comment:
It is better to test new hidet operator (under hidet.graph.ops) in ``tests/graph/operators`` in this way. 

To test the translation of pytorch model to hidet graph, it is better to
1. define a torch model (in this case, a model with a single operator that uses torch.nn.HardSigmoid),
2. run the model with hidet dynamoc backend,
3. compare the result with pytorch eager mode.